tests/fake_backends/test_fake_adonis.py

```diff
@@ -25,9 +26,13 @@ def test_iqm_fake_adonis():
     assert backend.name == 'IQMFakeAdonisBackend'
 
 
-def test_iqm_fake_adonis_connectivity(adonis_coupling_map):
+def test_iqm_fake_adonis_connectivity():
```

Comment:
Why is this no longer refering to the `adonis_coupling_map`?

Reply:
The fixture is only used here, so it was moved here. I did the same for Apollo and Aphrodite.

Reply:
It was also used in one other test, but it did not make much sense IMO, so I changed it.